src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -614,6 +616,12 @@ def transform_manifest(self, format: str, filters: JSON):
             response = es_search.execute()
             aggregate = response.aggregations
             manifest_config = self.generate_full_manifest_config(aggregate)
+            hit = response.hits.hits[0]
```

Comment:
This only works by accident, see https://github.com/DataBiosphere/azul/issues/1219. We may have to apply the header to the object in a separate request at the end, after the object has been created. Also, the full manifest can be generated for any filter. The response could contain any number of projects. Taking the one in the first hit isn't going to work.